test/powershell/engine/DefaultCommands.Tests.ps1

```diff
@@ -0,0 +1,497 @@
+"Cmdlet", "Write-Error",                                        ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"Cmdlet", "Write-EventLog",                                     ,                     $($FullCLR                               )
+"Cmdlet", "Write-Host",                                         ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"Cmdlet", "Write-Information",                                  ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"Cmdlet", "Write-Output",                                       ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"Cmdlet", "Write-Progress",                                     ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"Cmdlet", "Write-Verbose",                                      ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"Cmdlet", "Write-Warning",                                      ,                     $($FullCLR -or $CoreWindows -or $CoreUnix)
+"@
+
+            $aliaslist  = $commandList | ConvertFrom-CSV -Delimiter "," | Where-Object { $_.Present -eq "True" -and $_.CommandType -eq "Alias"  } | ForEach-Object { "{0} -> {1}" -f $_.Name, $_.Definition}
+            $cmdletlist = $commandList | ConvertFrom-CSV -Delimiter "," | Where-Object { $_.Present -eq "True" -and $_.CommandType -eq "Cmdlet" } | Select-Object -ExpandProperty Name
+
+            # We control only default engine aliases (Source -eq "") and aliases from following default loaded modules
+            # We control only default engine Cmdlets (Source -eq "") and Cmdlets from following default loaded modules
+            $moduleList = @("Microsoft.PowerShell.Utility", "Microsoft.PowerShell.Management", "Microsoft.PowerShell.Security", "Microsoft.PowerShell.Host", "Microsoft.PowerShell.Diagnostics", "PSWorkflow", "Microsoft.WSMan.Management", "Microsoft.PowerShell.Core")
+            Import-Module -Name $moduleList -ErrorAction SilentlyContinue
+
+            # We would use the session state to get default alias and cmdlet list
+            # instead of Get-Alias and Get-Command
+            # but latest is more simple use.
+            #$iis = [initialsessionstate]::CreateDefault()
+            #$currentCmdletList = $iis.Commands | Where-Object { $_.CommandType -eq "Cmdlet"} | Select-Object -ExpandProperty Name
+    }
+
+    It "All approved aliases present (no aliases removed, no new aliases added)" {
+        $currentAliasList = Get-Alias | Where-Object { $_.Source -eq "" -or $moduleList -contains $_.Source } | Select-Object -ExpandProperty DisplayName
+        $result = Compare-Object -ReferenceObject $currentAliasList -DifferenceObject $aliaslist
+        # Below 'Should Be' don't show full list wrong aliases so we output them explicitly
+        # if all aliases is Ok we output nothing
+        $result | Write-Host
+        $result | Should Be $null
+    }

```

Comment:
Yes, I'll remove the duplication. I leave it for easy review and for checking new tests in DefaultCommands.Tests.ps1.